mappings/net/minecraft/client/gl/IGLUniform.mapping

```diff
@@ -0,0 +1,28 @@
+CLASS ckp net/minecraft/client/gl/IGLUniform
```

Comment:
Likewise, GlUniform. We don't capitalize both letters, generally. (Some cases of UUID -> Uuid have to be fixed)